src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -359,6 +359,16 @@ TEST(OVClassBasicTest, smoke_SetConfigAutoNoThrows) {
     EXPECT_EQ(value, ov::hint::Priority::HIGH);
 }
 
+TEST(OVClassBasicTest, smoke_SetConfigWithSecondaryPropertiesThrows) {
+    ov::Core ie = createCoreWithTemplate();
+    ASSERT_THROW(ie.set_property(CommonTestUtils::DEVICE_CPU,
+                                 {ov::device::properties(CommonTestUtils::DEVICE_CPU, ov::num_streams(20))}),
+                 ov::Exception);
```

Comment:
Sorry for misleading you. secondary property setting means setting device properties via `ov::device::properties` within `set_property()`.
@wangleis , could you help explain why to prohibit device configuration via `ov::device::properties` through `set_property()`?

Reply:
@Wovchena The `ov::device::properties` support in `core::set_properties()` is WA for benchmark app to avoid breaking CI validation. Because current benchmark app use `core::set_properties()` to set properties include secondary properties. 

After discussion with Ilya Lavrenov and Ilya Churaev, we agree to remove this WA and update benchmark app by using `compile_model()` to set properties.